src/main/70-operational-database.tf

```diff
@@ -0,0 +1,30 @@
+module "aurora_postgresql_v2" {
+  source = "terraform-aws-modules/rds-aurora/aws"
+
+  name              = "${var.app_name}-operational-database-${var.env}"
+  engine            = "aurora-postgresql"
+  engine_mode       = "provisioned"
+  engine_version    = "14.6"
+  storage_encrypted = true
+
+  vpc_id                = module.vpc.vpc_id
+  subnets               = module.vpc.database_subnets
+  create_security_group = true
+  allowed_cidr_blocks   = module.vpc.private_subnets_cidr_blocks
```

Comment:
I'm thinking we could allow access by using EKS nodes SG instead of CIDR. Do you see any issues with that approach? 🤔

Reply:
No problem at all, I can open an issue to resolve when EKS module will be merged